url/extname.ts

```diff
@@ -10,5 +11,6 @@ import { posixExtname } from "../path/_extname.ts";
  */
 export function extname(url: string | URL) {
   url = new URL(url);
-  return posixExtname(url.pathname);
```

Comment:
I think this was more appropriate

Reply:
The only difference in new vs old is, in the new one if the url is like `deno.land`, it can extract `.land` from it which i believe can be a useful feature.

Reply:
> in the new one if the url is like deno.land, it can extract .land from it

I don't think that's what users expect as `.land` is top level domain, not an extension of the url.

Reply:
Done.